src/main/java/mate/academy/service/impl/ParseFileImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.model.FruitTransaction;
+import mate.academy.service.ParseFile;
+
+public class ParseFileImpl implements ParseFile {
+    private static final int INDEX_TYPE_OPERATION = 0;
+    private static final int INDEX_FRUIT = 1;
+    private static final int INDEX_QUANTITY = 2;
+
+    public List<FruitTransaction> parseFile(List<String> records) {
+        return records.stream()
+                .skip(1)
+                .map(line -> line.trim().split(","))
+                .map(splitLine -> new FruitTransaction(splitLine[INDEX_TYPE_OPERATION],
+                        splitLine[INDEX_FRUIT],
+                        Integer.parseInt(splitLine[INDEX_QUANTITY])))
+                .map(this::checkRecords)
+                .collect(Collectors.toList());
+    }
+
+    private FruitTransaction checkRecords(FruitTransaction fruitTransaction) {
+        List<String> listOfTypeOperations = new ArrayList<>();
+        for (FruitTransaction.Operation operation : FruitTransaction.Operation.values()) {
+            listOfTypeOperations.add(operation.getOperations());
+        }
```

Comment:
```suggestion
               List<String> listOfTypeOperations = Arrays.stream(Operation.values())
            .map(Enum::name)
            .collect(Collectors.toList());
```

can we do smth like this?